src/components/User/Operations/OperationDetails.vue

```diff
@@ -12,12 +14,63 @@
 <script>
 import DetailsReader from '../../common/details/DetailsReader'
 
+// ToDo: get operation by operationId from server
+import moment from 'moment'
+import get from 'lodash/get'
+import { ErrorHandler } from '@/utils/ErrorHandler'
+import { ApiCallerFactory } from '@/api-caller-factory'
+import { clearObject } from '@/utils/clearObject'
+
 export default {
   components: {
     DetailsReader,
   },
   props: {
-    operation: { type: Object, required: true },
+    operationId: { type: String, required: true },
+  },
+  data: _ => ({
+    userId: null,
+    operation: {},
+  }),
+  async created () {
+    window.scroll(0, 0)
+    this.userId = this.$route.params.id
+    await this.getOperations(this.operationId)
+  },
+  methods: {
+    async getOperations (operationId) {
+      let response = {}
+      try {
+        response = await ApiCallerFactory
+          .createCallerInstance()
+          .getWithSignature('/v3/history', {
+            page: { order: 'desc' },
+            filter: clearObject({
+              account: this.userId,
+            }),
+            include: ['operation.details'],
+          })
+      } catch (error) {
+        ErrorHandler.processWithoutFeedback(error)
+      }
+      response.data.filter(item => {
```

Comment:
Move filter result to the variable and let's work with this variable instead of `item`
It can save us from useless nesting and will make the code more readable